docs/geedocs/answer/2987876.html

```diff
@@ -4,59 +4,63 @@
-<h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Install GEE Server</h1>
+<h1><a href="../index.html">Google Earth Enterprise Documentation Home</a></h1>
 <h2>Legal notices</h2>
 
-<p>Copyright © 2008, 2009, 2010, 2011, 2012, 2013, 2014 Google, Inc.<br/>
+<p>Copyright &copy; 2008-2017 Google, Inc.<br/>
+Copyright &copy; 2017 Open GEE Contributors<br/>
 All rights reserved.</p>
-<p>Google, Google Earth, and Keyhole are trademarks or registered trademarks of Google, Inc. in the United States and elsewhere. Other trademarks not owned by Google, Inc. are the property of their respective owners.</p>
-<p>All Google and Keyhole logos and graphics are copyrighted. The content of this publication is furnished for informational use only, is subject to change without notice, and should not be construed as a commitment by Google, Inc.</p>
-<p>This publication is provided pursuant to and subject to the terms of Google Earth Enterprise. This publication and the information herein may not be reverse engineered, reproduced, duplicated, modified, altered, disclosed, published, printed, or distributed to any third party whatsoever without the express written permission of Google, Inc.</p>
-<p>Google Earth Enterprise may utilize the following products as integrated components, linked libraries, or optional modules. No usage rights, expressed or implied, are provided for said products independent of Google Earth Enterprise, unless otherwise stated in the product documentation. All trademarks are the property of their respective owners.</p>
+<p>
+  Google, Google Earth, and Keyhole are trademarks or registered trademarks of Google, Inc. in the United States and
+  elsewhere. Other trademarks not owned by Google, Inc. are the property of their respective owners.
+</p>
+<p>
+  All Google and Keyhole logos and graphics are copyrighted. The content of this publication is furnished for 
+  informational use only, is subject to change without notice, and should not be construed as a commitment by 
+  Google, Inc. or Open GEE Contributors.
+</p>
+<p>
+  Open Google Earth Enterprise is licensed under the 
+  <a href="https://raw.githubusercontent.com/google/earthenterprise/master/LICENSE">Apache License v2.0</a>
+</p>
+<p>
+  Open Google Earth Enterprise may utilize the following products as integrated components, linked libraries, or
+  optional modules. No usage rights, expressed or implied, are provided for said products independent of Open Google 
+  Earth Enterprise, unless otherwise stated in the product documentation. All trademarks are the property of their
+  respective owners. Each product is released under their own respective license. For bundled products, license 
+  information is available under the 
+  <a href="https://github.com/google/earthenterprise/tree/master/earth_enterprise/third_party">third_party directory</a>
+  of the repository.
+</p>
```

Comment:
Does the first phrase '.... may utilize the following products'  refers to the un-bundled products (that are now unlisted) as later we say 'For bundled products, license information...' ? If yes, may want to remove 'the following' so it read '...may utilize products as ...'.

Reply:
I don't understand the reasoning for the request. I can remove the word "may". In all likelihood it will utilize this list. I guess I can say "utilizes the following bundled products"

Reply:
yes: i would remove 'may' as these products are bundled. I believe 'may' was referring to the other products  (like MrSID, etc.) that you removed.

Reply:
OK, updated